src/Application/ApplicationFileProcessor.php

```diff
@@ -113,4 +127,34 @@ private function printFile(File $file): void
         $this->smartFileSystem->dumpFile($smartFileInfo->getPathname(), $file->getFileContent());
         $this->smartFileSystem->chmod($smartFileInfo->getRealPath(), $smartFileInfo->getPerms());
     }
+
+    /**
+     * Inspired by @see https://github.com/phpstan/phpstan-src/blob/89af4e7db257750cdee5d4259ad312941b6b25e8/src/Analyser/Analyser.php#L134
+     */
+    private function configureCustomErrorHandler(): void
+    {
+        $errorHandlerCallback = function (int $code, string $message, string $file, int $line): bool {
+            error_reporting();
+            if ((error_reporting() & $code === 0) !== 0) {
```

Comment:
unneeded call and repetitive parentheses can be removed.

```suggestion
+           if ((error_reporting() & $errorType) === 0) {
```